src/borg/archiver.py

```diff
@@ -2237,7 +2237,7 @@ def do_break_lock(self, args, repository):
         File patterns support these styles: fnmatch, shell, regular expressions,
         path prefixes and path full-matches. By default, fnmatch is used for
         ``--exclude`` patterns and shell-style is used for the ``--pattern``
-         option. For commands that support patterns in their ``PATH`` argument
```

Comment:
Regression introduced in 5d4dda75d36d4bdaff9fc4a38a234383aef4b733 - nothing to backport

Reply:
yup.